tests/sanity_helpers.py

```diff
@@ -148,9 +148,27 @@ def __init__(self):
         self.pvc_objs = list()
         self.pod_objs = list()
         self.ceph_cluster = CephClusterExternal()
+
+    def create_resources(self, pvc_factory, pod_factory):
```

Comment:
can we add it also to tests.sanity_helpers.Sanity.create_resources?

Reply:
@ebenahar , I didn't get it completely. You meant to move obc deletion to parent class ?

Reply:
I meant that the new functionalities added here would be relevant for internal mode cluster as well